PanelViewController/Framework source/OverlayView.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  OverlayView.swift
+//  PanelViewController
+//
+//  Created by Rynn, David on 5/3/18.
+//  Copyright © 2018 NYC DoITT. All rights reserved.
+//
+
+import UIKit
+
+class OverlayView: UIView {
```

Comment:
Do you think there's a way to name this class so that we know it ignores touches and passes them on? Or maybe just add a comment.